apiserver/pkg/manager/resource_manager.go

```diff
@@ -63,7 +71,7 @@ func (r *ResourceManager) getKubernetesNamespaceClient() clientv1.NamespaceInter
 // clusters
 func (r *ResourceManager) CreateCluster(ctx context.Context, apiCluster *api.Cluster) (*v1alpha1.RayCluster, error) {
 	// populate cluster map
-	computeTemplateDict, err := r.populateComputeTemplate(ctx, apiCluster)
```

Comment:
just like to confirm, these refactor are just for reuse, right?

Reply:
yes